src/components/PageHeader.tsx

```diff
@@ -18,16 +18,27 @@ const StyledHeader = styled.div`
 const StyledTypography = styled(Typography)`
   border-bottom: 3px solid;
   padding-bottom: 0.5rem;
+  cursor: pointer;
```

Comment:
er det ikke litt rart om det blir en pointer også for korte headers som ikke er kortet ned? lurer på om det bare burde gjelde for `<TextTruncate>`

Reply:
oh, godt poeng!

Reply:
ah, den får pointer automatisk pga onClicken på Typography...

Reply:
og hvis tittelen ikke kan klikkes på når den er expanded så kan man ikke toggle. Så tror det er best å la det være sånn som det er nå

Reply:
🤯